bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'activejob/limiter'
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+# require 'pry'
+# Pry.start
+
+require 'irb'
+IRB.start(__FILE__)
```

Comment:
🔥 these unneeded scaffold files?

Reply:
They're generated by Bundler when you make a new gem. They do provide functionality so I didn't think it was necessary to delete them.